src/components/modeler/XMLManager.js

```diff
@@ -0,0 +1,51 @@
+import runningInCypressTest from '@/runningInCypressTest';
+import FilerSaver from 'file-saver';
+
+export default class XMLManager {
+  #moddle;
+  #definitions;
+
+  constructor(moddle) {
+    this.#moddle = moddle;
+  }
+
+  set definitions(definitions) {
+    this.#definitions = definitions;
+  }
+
+  getDefinitionsFromXml(xmlString) {
+    return new Promise((resolve, reject) => {
+      this.#moddle.fromXML(xmlString, (err, definitions) => {
+        if (err) {
+          reject(err);
+        }
+
+        definitions.exporter = 'ProcessMaker Modeler';
```

Comment:
I'm wondering if we should move these types of strings to a central place (thing of my abuse of the constants file in the `e2e` directory as well). I know the same string is in `ModelerApp.vue`, for instance.

Also, should this string be passed through `$t()` here or is this always going to be an  untranslated exporter value?

Reply:
If `definitions.exporter` is suppose to be the name of the app, and be kept in sync with the page title in `ModelerApp.vue`, then yes, I think it makes sense to pull out the string and keep it in a single place.

However, core currently uses a different `ModelerApp.vue` file and has no reference to a "ProcessMaker Modeler" string. I'm also not sure how this string is used or if it should be translated. I'll clarify with Alan, but I think it's fine keeping it the way it is for now.

Reply:
👍

Reply:
Looking at the spec it's support to identify the name and version of the app that produced the XML:

<img width="649" alt="Screen Shot 2020-02-20 at 2 15 26 PM" src="https://user-images.githubusercontent.com/7561061/74969940-84e15000-53eb-11ea-967f-279cf51f1f70.png">

So I don't think these values should be translated, but it might be worth pulling them from our `package.json` file to help with debugging. That way, if we come across an issue and have the XML, we can see something like `exporter="@processmaker/modeler" exporterVersion="0.18.4"`.

@velkymx thoughts on this?